src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -202,12 +217,19 @@ hsModuleForDotProto
 
        let hasService = has (traverse._DotProtoService) protoDefinitions
 
-       let importDeclarations = concat [ defaultImports hasService, extraImports, typeContextImports ]
+       let importDeclarations = concat
+              [ defaultImports
+                ImportCustomisation
+                  { usesGrpc = hasService
+                  , ..
```

Comment:
`RecordWildCards` are useful in many contexts, but perhaps they obscure things here?  I would prefer using `NamedFieldPuns` and mentioning the field to be copied from the argument by the same name.

Reply:
There was a name conflict issue; I prefixed the field name instead

Reply:
OK.